### ISSUE TITLE
basic_info: sso related info not successfully retrieved

### ISSUE THREAD

**Issue by kmahyyg** [kmahyyg/DFIR4vSphere-go#2]
![20230128-124319-Jump Desktop-001360](https://user-images.githubusercontent.com/16604643/215242428-1e740c66-0d18-4684-9d32-ab4e40310fe1.png)

**Comment by kmahyyg** [comment]
```json
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api/vc_basic_info.go:164","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api.(*vSphereClient).ListPermissions","level":"debug","msg":"permission list length is not zero.","time":"2023-01-28T12:43:04.9044065+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api/vc_basic_info.go:173","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api.(*vSphereClient).ListPermissions","level":"debug","msg":"executed: transform permissions object in ListPermissions","time":"2023-01-28T12:43:04.9755186+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds/basic_info.go:39","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds.RetrieveBasicInformation","level":"info","msg":"list permission finished.","time":"2023-01-28T12:43:04.9760144+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api/vc_basic_info.go:214","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api.(*vSphereClient).ListAllUsers","level":"debug","msg":"executed: created sso admin client in ListAllUsers","time":"2023-01-28T12:43:05.0463608+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api/vc_basic_info.go:219","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/vsphere_api.(*vSphereClient).ListAllUsers","level":"error","msg":"get identity sources err.","time":"2023-01-28T12:43:05.061741+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds/basic_info.go:43","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds.RetrieveBasicInformation","level":"error","msg":"list all users, err:  ServerFaultCode: NotAuthenticated","time":"2023-01-28T12:43:05.062254+08:00"}
{"file":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds/basic_info.go:45","func":"github.com/kmahyyg/DFIR4vSphere-go/pkg/subcmds.RetrieveBasicInformation","level":"info","msg":"list all users finished.","time":"2023-01-28T12:43:05.0627333+08:00"}
```

**Comment by kmahyyg** [comment]
fixed by e533f16 and f3c6ac8
